packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -528,7 +551,7 @@ export default class Editor implements IEditor {
         nameOrMap: string | Record<string, DOMEventHandler>,
         handler?: DOMEventHandler
     ): () => void {
-        const eventsToMap = typeof nameOrMap == 'string' ? { [nameOrMap]: handler } : nameOrMap;
+        const eventsToMap = typeof nameOrMap == 'string' ? { [nameOrMap]: handler! } : nameOrMap;

```

Comment:
Can handler really be null or undefined?

Reply:
According to Typescript, no, as the overload either doesnt have a string `nameOrMap` or both parameters are required. However one might still introduce a nullish handler by using ts-ignore or regular JS. I can add a runtime check just in case.